prow/config/config.go

```diff
@@ -128,6 +129,14 @@ type Plank struct {
 	Controller `json:",inline"`
 }
 
+// Gerrit is config for the gerrit controller.
+type Gerrit struct {
+	// TickInterval is how ofen we do a sync with binded gerrit instance
+	TickInterval time.Duration `json:"tick_interval,omitempty"`
```

Comment:
I think you need to make this a pointer if you want omitempty to work.

Reply:
added `TickIntervalString`